integration/experiment/launch_util.py

```diff
@@ -70,7 +70,7 @@ def launch_run(agent_conf, app_conf, run_id, output_dir, extra_cli_args,
     bash_path = apps.make_bash(app_conf, run_id, log_path)
     argv.extend([bash_path])
 
-    num_ranks = app_conf.get_rank_per_node() * num_nodes
+    num_ranks = app_conf.get_total_ranks(num_nodes)
```

Comment:
This change will break all the apps that do not implement get_total_ranks.  Unclear just now why this was required to change.